meow/visualize.py

```diff
@@ -46,10 +46,14 @@ def _visualize_s_matrix(S, fmt=".3f", title=None):
     plt.ylim(coords_[-1] - 0.5, coords_[0] + 0.5)
     plt.grid(True)
 
-    for x, y, z in zip(X.ravel(), Y.ravel(), Z[::-1].T.ravel()):
-        if z > 0.0005:
+    for x, y, z in zip(X.ravel(), Y.ravel(), S[::-1].T.ravel()):
+        if np.abs(z) > 0.0005:
             text = eval(f"f'{{z:{fmt}}}'")  # 😅
-            plt.text(x, y, text, ha="center", va="center")
+            text = text.replace("+", "\n+")
```

Comment:
maybe we can show complex numbers as absolute value and phase in stead of real and imag?

Reply:
jep we could do that. I think that would make sense. My modification to visualize was just a quick hack to get the information I needed